packages/server/tests/contracts/contracts_v2/BaseHTS.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "../FeeHelper.sol";
+
+contract BaseHTS is FeeHelper {
```

Comment:
I never really liked the name `BaseHTS.sol` if you're splitting the contracts up then let's match the Smart-Contracts repo. 
So instead of a v2 just start a `TokenCreate.sol` with this.
We can then break `BaseHTS.sol` into `TokenCreateContract.sol` and `TokenManagementContract.sol` later

For the acceptance tests tag you could add a `@tokencreate` label
e.g. `describe('@tokencreate HTS Precompile Token Create Acceptance Tests'`